replication_chexnet_cuda_local.py

```diff
@@ -289,17 +307,20 @@ def eval_model(model, test_loader, logfile, setstr):
 # CUDA stats
 print(torch.cuda.device_count())
 print(torch.cuda.get_device_name(0))
-
 # cudnn will look for the optimal set of algorithms for that particular configuration (which takes some time). 
 # This usually leads to faster runtime.
 cudnn.benchmark = True
 
-# initialize and load the model
+
+""" Initialized the Model
+
+If there is a trained model, it can be loaded.
+"""
 model = DenseNet121(N_LABEL).cuda()
-# load trained model if needed
-model.load_state_dict(torch.load("20trained.pth"))
+# model.load_state_dict(torch.load("trained.pth"))
 
 
+""" Initialize the Dataset"""
 train_dataset = XrayDataSet(DATA_PATH, "final_train.txt", train_sampling=False)
```

Comment:
this `train_sampling=False` could be removed, I believe it was added by @lytinahome for testing purpose

Reply:
cleaned